index.html

```diff
@@ -27,9 +27,115 @@
         <a id="collaboration">
             Start collaboration
         </a>
-    </headline>
+    </h1>
 </head>
 <body>
-    
+    <h2 class="h2">
+        Projects
+    </h2>
+    <div class="grid-container">
```

Comment:
- [ ] great job. kindly make the main container of the works section is  a `section tag`

Reply:
Hi! What do you mean? can you explain further